pages/index.js

```diff
@@ -20,45 +20,128 @@ export default function Home(props) {
+                  {t("circleTxt1/4")}
+                </p>
+              </div>
+            </div>
+            <div className="flex flex-row ">
+              <div
+                className="bg-circle-color flex-shrink-0 opacity-60 mr-4 mb-2  rounded-full h-32 w-32 flex items-center justify-center
+           text-white font-bold text-h1xl  relative md:left-0 -left-14 "
+              >
+                2{" "}
+              </div>
+              <div className=" self-center -ml-14 md:ml-0">
+                <p className="text-sm md:text-p leading-normal text-left font-normal font-body ">
+                  {t("circleTxt2/4")}
+                </p>
+              </div>
+            </div>
+            <div className="flex flex-row ">
+              <div
+                className="bg-circle-color flex-shrink-0 opacity-80  mr-4 mb-2 rounded-full h-32 w-32  flex items-center justify-center
+           text-white font-bold text-h1xl  relative md:left-0 -left-14"
+              >
+                3{" "}
+              </div>
+              <div className=" self-center -ml-14 md:ml-0">
+                <p className="text-sm md:text-p leading-normal text-left font-normal font-body">
+                  {t("circleTxt3/4")}
+                </p>
+              </div>
+            </div>
+            <div className="flex flex-row ">
+              <div
+                className="bg-circle-color flex-shrink-0 opacity-100  mr-4 mb-2  rounded-full h-32 w-32 flex items-center justify-center
+           text-white font-bold text-h1xl  relative md:left-0 -left-14"
+              >
+                4{" "}
+              </div>
+              <div className=" self-center -ml-14 md:ml-0">
+                <p className="text-sm md:text-p leading-normal text-left font-normal font-body">
+                  {t("circleTxt4/4")}
+                </p>
+              </div>
+            </div>
+          </div>
 
-      <section className="layout-container">
-        <h4>{t("featuredPrototypesHeading")}</h4>
-        <figure>Block</figure>
-        <figure>Block</figure>
-        <figure>Block</figure>
+          <div>
+            <h1>{t("aboutTitle")}</h1>
+            <br></br>
+            <p className="text-sm md:text-p  leading-normal text-left font-body  font-normal">
+              {t("AboutThisSite1/3")}
+              <br></br>
+              <br></br>
+              {t("AboutThisSite2/3")}
+              <br></br>
+              <br></br>
+              {t("AboutThisSite2/3")}
+            </p>
+            <br></br>
+            <div className="invisible xl:visible">
+              <ActionButton
+                text={t("learnMoreBtn")}
+                className={"text-xs md:text-base"}
+                secondary
+                dataCyButton={"learn-more"}
+              />
+            </div>
+          </div>
+        </div>
+        <div className="xl:invisible bg-white md:pl-16 lg:pl-32  pt-6 xl:pt-0 xl:pb-0 pb-10 pl-6">
+          <ActionButton
```

Comment:
This looks kind of unaligned with the rest of the pages content alignment at certain sizes. Like here:

![tempsnip](https://user-images.githubusercontent.com/31868510/115575938-0737ff00-a291-11eb-9b72-fdf16235f4fb.png)

It is also slightly off at smaller sizes. Should just be able to use the layout-container instead of the left padding.